pkg/build/builder/daemonless.go

```diff
@@ -201,9 +201,11 @@ func pullDaemonlessImage(sc types.SystemContext, store storage.Store, imageName
 		BlobDirectory: blobCacheDirectory,
 	}
 	_, err = buildah.Pull(context.TODO(), "docker://"+imageName, options)
-	log.V(0).Infof("Finished pulling image %s", imageName)
+	if err == nil {
+		log.V(2).Infof("Finished pulling image %q", imageName)
+	}
 	if err != nil && dockerConfigCredsErr != nil {
-		err = fmt.Errorf("%s; also, error processing dockerconfigjson: %s", err.Error(), dockerConfigCredsErr.Error())
+		err = fmt.Errorf("Error pulling image %q: %s; also, error processing dockerconfigjson: %s", imageName, err.Error(), dockerConfigCredsErr.Error())
```

Comment:
sorry i missed this earlier..... we only return this error if there is both a configjson error *and* an error pulling?  that seems weird.  (Though i realize it's not changed by this PR)

Also since i had you add the conditional to only print the finished statement when err==nil, i guess we also need to log something explicitly to demarcate completion when err!=nil.  (Rather than just relying on the caller to print out the error that we are returning here)

Reply:
yeah weird but there is some reasoning behind it as I recall ... unexpected errors because of bad pull secrets vs. an error which we might be able to ignore if we retry with the node credentials

after I pull in your suggested error i'll add a conditional to cover this gap